test/test_ops.py

```diff
@@ -192,6 +228,48 @@ def wrapped_fn(*args, **kwargs):
 
             self.assertEqual(result, expected)
 
+    @ops(functorch_lagging_op_db + additional_op_db, allowed_dtypes=(torch.float,))
+    @skipOps('TestOperators', 'test_jvp', set({
+        # Apprently these support forward AD, but we get "Trying to use forward AD..."
+        xfail('var_mean'),
+        xfail('std_mean'),
+
+        # RuntimeError: expected scalar type double but found float
+        xfail('minimum'),
+        xfail('min', 'binary'),
+        xfail('maximum'),
```

Comment:
@soulitzer I dug into this a little. "expected scalar type double but found float" is a bug with PyTorch forward-mode AD, not functorch. We should leave a comment here to that effect (or fix the forward formula for these)
```
import torch
import torch.autograd.forward_ad as fwAD

x = torch.randn(3)
y = torch.randn(3)
tx = torch.randn(3)
ty= torch.randn(3)

with fwAD.dual_level():
  x_dual = fwAD.make_dual(x, tx)
  y_dual = fwAD.make_dual(y, ty)
  torch.maximum(x_dual, y_dual)
```